src/components/TopBar/components/Nav.tsx

```diff
@@ -1,28 +1,69 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { keyframes } from "styled-components";
 import StyledRouterLink from "./StyledRouterLink";
 import StyledLink from "./StyledLink";
-import StyledNestedLink from "./StyledNestedLink";
 
-const Nav: React.FC = () => {
+interface NavProps {
+  onDismiss: () => void;
+  mobileMenu?: boolean;
+}
+
+const Nav: React.FC<NavProps> = ({ onDismiss, mobileMenu }) => {
   return (
-    <StyledNav>
-      <StyledRouterLink target="/dashboard" label="Dashboard" />
-      <StyledNestedLink target="/umbrella" label="Projects" type="router" >
-        <StyledRouterLink target="/umbrella" label="Umbrella" style={{ height: 26, display: "flex", alignItems: "center" }} />
-        <StyledLink href="https://degenerative.finance/" label="Degenerative" style={{ height: 26, display: "flex", alignItems: "center" }} />
-      </StyledNestedLink>
-      <StyledRouterLink target="/governance" label="Govern" />
-      <StyledRouterLink target="/farm" label="Farm" />
-      <StyledRouterLink target="/migrate" label="Migrate" />
-      <StyledLink href="https://docs.yam.finance/" label="FAQ" />
-    </StyledNav>
+    <>
+      { mobileMenu && (
+        <StyledBackdrop onClick={onDismiss} />
+      )}
+      <StyledNav>
+        <StyledRouterLink target="/dashboard" label="Dashboard" mobileMenu={mobileMenu} onDismiss={onDismiss} />
+        <StyledRouterLink target="/umbrella" label="Projects" mobileMenu={mobileMenu} onDismiss={onDismiss}>
+          <StyledRouterLink target="/umbrella" label="Umbrella" mobileMenu={mobileMenu} onDismiss={onDismiss} />
+          <StyledLink href="https://degenerative.finance/" label="Degenerative" mobileMenu={mobileMenu} onDismiss={onDismiss} />
+        </StyledRouterLink>
```

Comment:
in the smaller devices view, once you click on a menu with sub pages, we need it to look like this:
![1](https://user-images.githubusercontent.com/2432272/108562339-252cc600-7319-11eb-8e1a-22d876a7d3ac.PNG)
- adding more space on the left side for the sub labels
- updating the color of the background of the divider, to be similar to the color that appears on the text at the umbrella page "Coming Soon!" with 33% transparency `rgb(XXX XXX XXX / 33%)`